tests/selection.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+import skcosmo
+from sklearn.datasets import load_boston
+
+
+class PCovSelectionTest:
+    def test_no_Y(self):
+        X, Y = load_boston(return_X_y=True)
+
+        with self.assertRaises(Exception):
```

Comment:
You can check the message (to ensure that the error the test see is the one you think the test see) while still using the context manager version of `assertRaises` with 
```suggestion
        with self.assertRaises(Exception) as cm:
             # XXX
         self.assertEqual(str(cm.exception), "the error message")
```

Reply:
If the error come from code you wrote, I would say yes!